### PR TITLE
Adding Homework 5 for Victor Dimitrov

### PR DESCRIPTION
**Pull request by Vic-Dim**

### CODE REVIEW

File: homework_05/04_victor_dimitrov/decrypt.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+            decrypted_letter -= key
+
+            if not (LOWERCASE_A <= decrypted_letter <= LOWERCASE_Z):
+                decrypted_letter += ALPHABET_LENGTH
+
+
+
+        decrypted_word += chr(decrypted_letter)
+
+    return decrypted_word
```
Comment (Teodor1331) [inline homework_05/04_victor_dimitrov/decrypt.py:92]: The reversed encrypting function for decrypting here is right.

File: homework_05/04_victor_dimitrov/decrypt.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+        tokenizer = RegexpTokenizer(r'\w+')
+        lemma = WordNetLemmatizer()
+
+        word_token = tokenizer.tokenize(word)[0]
+        lemma_word = lemma.lemmatize(tokenizer.tokenize(word)[0])	
+
+        if not wordnet.synsets(lemma_word) and word_token.lower() not in words.words():
+            return False
+
+    return True
```
Comment (Teodor1331) [inline homework_05/04_victor_dimitrov/decrypt.py:67]: I like the idea of checking the message for valid words. Very good!

**Review by Teodor1331 — APPROVED** [review]
Good job! The program foresee for some special cases. Excellent!

File: homework_05/04_victor_dimitrov/decrypt.py (hunk 1 above)
Comment (StelianRBG) [reply homework_05/04_victor_dimitrov/decrypt.py:92]: Good idea for function

File: homework_05/04_victor_dimitrov/decrypt.py (hunk 2 above)
Comment (StelianRBG) [reply homework_05/04_victor_dimitrov/decrypt.py:67]: I like you split your code on functions

File: homework_05/04_victor_dimitrov/decrypt.py [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+                if not are_valid:
+                    decrypted_words = []
+                    continue
+
+                else:
+                    right_key = key
+
+
+        if right_key != 0:
+            break
```
[the remark this answers is not in the document]
Comment (StelianRBG) [reply homework_05/04_victor_dimitrov/decrypt.py:35]: For key you can only write in range(26)

**Review by StelianRBG — COMMENTED** [review]
Good solution

**Review by Vladikolev0321 — COMMENTED** [review]
Good job.

**Pull request merged**